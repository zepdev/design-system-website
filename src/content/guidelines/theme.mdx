---
label: Guidelines
title: Theme
---

# Theme

Themes are used to customize existing elements to fit the specific style of a strategic business unit or product. There is no base color for the Zeppelin-element-library, almost all elements require that a theme class is present. At the moment the Zeppelin Design System offers three main themes. Developers can customize all elements by adding the theme class to any html tag (`<main>`,`<body>`,`<div>`, ...) as long it wraps all elements you want to use.

```js
<main class="theme-zeppelin" id="app-root">
  ...
</main>
```

We currently offer three main themes;

```js
// Zeppelin Indigo Blue
<div class="theme-zeppelin" >...</div>

// CAT Yellow
<div class="theme-cat" >...</div>

// Rental Red
<div class="theme-rental" >...</div>
```
