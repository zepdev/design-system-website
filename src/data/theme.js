module.exports = {
  breakpoints: {
    xs: '36em' /* 576px */,
    s: '48em' /* 768px */,
    m: '62em' /* 992px */,
    l: '75em' /* 1200px */,
    xl: '105em' /* 1680px */,
    xxl: '120em' /* 1920px */,
  },
  typography: {
    typefaces: {
      roboto: "'Roboto', Arial, Helvetica, sans-serif",
      teko: "Teko', sans-serif",
    },
    basics: {
      fontSizeBase: {
        specification: 'font-size-base',
        px: '16px',
        rem: '1rem',
        weight: null,
      },
      lineHeightBase: {
        specification: 'line-height-base',
        px: '24px',
        rem: '1.5rem',
        weight: null,
      },
      fontWeightBase: {
        specification: 'font-weight-base',
        px: null,
        rem: null,
        weight: 400,
      },
    },
    headings: {
      normal: {
        'zep-typo--normal-1': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-2': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-3': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-4': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-5': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-6': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-7': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--normal-8': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
      },
      display: {
        'zep-typo--display-1': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--display-2': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--display-3': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--display-4': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
      },
      special: {
        'zep-typo--special-1': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--special-2': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--special-3': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
        'zep-typo--special-4': {
          rem: '0.75rem',
          px: '12px',
          weight: 400,
          specifications: [
            'font-family: "Roboto"',
            'font-weight: 400',
            'line-height: 1.5',
            'text-transform: "none"',
          ],
        },
      },
    },
  },
  colors: {
    primary: {
      indigoBlue: {
        name: 'indigo-blue',
        hex: '#27166f',
        rgb: '39/22/111',
        hsb: null,
      },
      catYellow: {
        name: 'cat-yellow',
        hex: '#f0b823',
        rgb: '240/184/35',
        hsb: null,
      },
      rentalRed: {
        name: 'rental-red',
        hex: '#af0e0e',
        rgb: '175/14/14',
        hsb: null,
      },
    },
    gray: {
      grayDark: {
        name: 'gray-dark',
        hex: '#101111',
        rgb: '16/17/17',
        hsb: null,
      },
      grayMidDark: {
        name: 'gray-mid-dark',
        hex: '#373d3f',
        rgb: '55/61/63',
        hsb: null,
      },
      gray: { name: 'gray', hex: '#464a4c', rgb: '70/74/76', hsb: null },
      grayLight: {
        name: 'gray-light',
        hex: '#6f7c80',
        rgb: '111/124/128',
        hsb: null,
      },
      grayLighter: {
        name: 'gray-lighter',
        hex: '#eceeef',
        rgb: '236/238/239',
        hsb: null,
      },
      grayMid: {
        name: 'gray-mid',
        hex: '#f2f3f4',
        rgb: '242/243/244',
        hsb: null,
      },
      grayLightest: {
        name: 'gray-lightest',
        hex: '#f7f7f9',
        rgb: '247/247/249',
        hsb: null,
      },
      white: { name: 'white', hex: '#ffffff', rgb: '255/255/255', hsb: null },
    },
    support: {
      alert: { name: 'alert', hex: '#ff6562', rgb: '255/101/98', hsb: null },
      warning: {
        name: 'warning',
        hex: '#f0ad4e',
        rgb: '240,/73/78',
        hsb: null,
      },
      success: { name: 'success', hex: '#5cb85c', rgb: '92/184/92', hsb: null },
      info: { name: 'info', hex: '#5bc0de', rgb: '91/192/222', hsb: null },
    },
  },
  spacing: {
    xxs: 4,
    xs: 8,
    s: 12,
    m: 16,
    l: 24,
    xl: 32,
    xxl: 48,
  },
}
