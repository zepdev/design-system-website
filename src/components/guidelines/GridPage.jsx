import React, { useState } from 'react'
import PropTypes from 'prop-types'
import withStyles from 'react-jss'
import Tabs from '../tabs/Tabs'
import Tab from '../tabs/Tab'
import classnames from 'classnames'
import ElementUsage from '../elements/ElementUsage'
import ReactPlayer from 'react-player'
import Table from '../table/Table'
import { grid } from 'zeppelin-element-library/bundle/themes/theme.json'

const styles = theme => ({
  tabs: {
    marginBottom: `${ theme.spacing.component.xxl.rem }rem`,
  },
  player: {
    marginBottom: `${ theme.spacing.component.xl.rem }rem`,
  },
  headline: {
    marginBottom: `${ theme.spacing.component.l.rem }rem`,
  },
})

function GridPage({ classes }) {
  const [value, setValue] = useState(0)

  function handleChange(event, newValue) {
    setValue(newValue)
  }

  return (
    <>
      <Tabs value={value} onClick={handleChange} className={classes.tabs}>
        <Tab label="Overview" />
        <Tab label="Usage" />
      </Tabs>
      {value === 0 && (
        <>
          <h2 className={classnames('zep-typo--normal-8', classes.headline)}>Basic Grid</h2>

          <p className={classnames('zep-typo--normal-3', classes.headline)}>
            Zeppelin uses a hybrid grid layout as demonstrated below
          </p>

          <div className={classes.player}>
            <ReactPlayer
              url="https://cdn-zel.zepdev.net/zel/animation_grid.mp4"
              playing
              width="1906"
              height="â€Š1218"
              loop
            />
          </div>

          <h3 className={classnames('zep-typo--normal-6', classes.headline)}>Basic Grid Data</h3>
          <Table
            title="basic grid data"
            header={['identifier', 'screensize', 'columns', 'margin', 'gutter']}
            content={{
              xxs: { identifier: 'xxs', ...grid.xxs },
              xs: { identifier: 'xs', ...grid.xs },
              s: { identifier: 's', ...grid.s },
              m: { identifier: 'm', ...grid.m },
              l: { identifier: 'l', ...grid.l },
              xl: { identifier: 'xl', ...grid.xl },
            }}
          />
        </>
      )}
      {value === 1 && <ElementUsage element="grid" />}
    </>
  )
}

GridPage.propTypes = {
  classes: PropTypes.object,
}

export default withStyles(styles)(GridPage)
