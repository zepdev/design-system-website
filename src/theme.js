import { createMuiTheme } from '@material-ui/core/styles'
import guidelines from '@zlab-de/zel-react/guidelines.json'

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: guidelines.theme.indigo.secondary,
      main: guidelines.theme.indigo.primary,
      dark: guidelines.theme.indigo.primary,
    },
    error: { main: guidelines.color.support.alert },
  },
  status: {
    gray: '#eceeef',
  },
  logo: guidelines.logo,
  blue: guidelines.theme.blue,
  indigo: guidelines.theme.indigo,
  red: guidelines.theme.red,
  yellow: guidelines.theme.yellow,
  guidelines: guidelines,
  space: guidelines.spacing.component,
  color: guidelines.color,
})

export default theme
