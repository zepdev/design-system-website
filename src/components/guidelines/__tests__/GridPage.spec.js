import React from 'react'
import { render } from 'react-testing-library'
import { ThemeProvider } from 'react-jss'
import theme from 'zeppelin-element-library/bundle/themes/theme.json'
import GridPage from '../GridPage'

describe('GridPage', () => {
  it('renders correctly', () => {
    const mockProps = {}
    const component = render(
      <ThemeProvider theme={theme}>
        <GridPage {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
