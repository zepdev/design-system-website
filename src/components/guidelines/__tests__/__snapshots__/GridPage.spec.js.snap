// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridPage renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Tabs-root-0-1-4 GridPage-tabs-0-1-1"
        data-testid="tabs"
      >
        <button
          class="Tab-root-0-1-5 Tab-selected-0-1-11 Tab-normal-0-1-6 ButtonBase-button-0-1-12"
        >
          <p
            class="Tab-label-0-1-8 Tab-selected-0-1-11 Tab-labelPrimary-0-1-9 zep-typo--normal-4"
          >
            Overview
          </p>
        </button>
        <button
          class="Tab-root-0-1-5 Tab-normal-0-1-6 ButtonBase-button-0-1-12"
        >
          <p
            class="Tab-label-0-1-8 Tab-labelPrimary-0-1-9 zep-typo--normal-4"
          >
            Usage
          </p>
        </button>
      </div>
      <h2
        class="zep-typo--normal-8 GridPage-headline-0-1-3"
      >
        Basic Grid
      </h2>
      <p
        class="zep-typo--normal-3 GridPage-headline-0-1-3"
      >
        Zeppelin uses a hybrid grid layout as demonstrated below
      </p>
      <div
        class="GridPage-player-0-1-2"
      >
        <div>
          <video
            autoplay=""
            loop=""
            preload="auto"
            src="https://cdn-zel.zepdev.net/zel/animation_grid.mp4"
            style="width: 100%; height: 100%;"
          />
        </div>
      </div>
      <h3
        class="zep-typo--normal-6 GridPage-headline-0-1-3"
      >
        Basic Grid Data
      </h3>
      <div
        class="Table-root-0-1-13"
      >
        <table
          class="Table-table-0-1-14"
        >
          <caption
            class="zep-visually-hidden"
          >
            basic grid data
          </caption>
          <thead
            class="Table-header-0-1-18"
          >
            <tr
              class="Table-tr-0-1-16"
            >
              <th
                class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
                scope="col"
              >
                identifier
              </th>
              <th
                class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
                scope="col"
              >
                screensize
              </th>
              <th
                class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
                scope="col"
              >
                columns
              </th>
              <th
                class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
                scope="col"
              >
                margin
              </th>
              <th
                class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
                scope="col"
              >
                gutter
              </th>
            </tr>
          </thead>
          <tbody
            class="Table-tbody-0-1-17"
          >
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                xxs
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                &lt;576px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                4
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                24px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                24px
              </td>
            </tr>
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                xs
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                ≥576px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                6
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
            </tr>
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                s
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                ≥768px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                6
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                48px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
            </tr>
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                m
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                ≥992px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                8
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                64px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
            </tr>
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                l
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                ≥1200px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                12
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                64px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
            </tr>
            <tr
              class="Table-tr-0-1-16"
            >
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
                scope="row"
              >
                xl
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                ≥1680px
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                12
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                auto
              </td>
              <td
                class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              >
                32px
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Tabs-root-0-1-4 GridPage-tabs-0-1-1"
      data-testid="tabs"
    >
      <button
        class="Tab-root-0-1-5 Tab-selected-0-1-11 Tab-normal-0-1-6 ButtonBase-button-0-1-12"
      >
        <p
          class="Tab-label-0-1-8 Tab-selected-0-1-11 Tab-labelPrimary-0-1-9 zep-typo--normal-4"
        >
          Overview
        </p>
      </button>
      <button
        class="Tab-root-0-1-5 Tab-normal-0-1-6 ButtonBase-button-0-1-12"
      >
        <p
          class="Tab-label-0-1-8 Tab-labelPrimary-0-1-9 zep-typo--normal-4"
        >
          Usage
        </p>
      </button>
    </div>
    <h2
      class="zep-typo--normal-8 GridPage-headline-0-1-3"
    >
      Basic Grid
    </h2>
    <p
      class="zep-typo--normal-3 GridPage-headline-0-1-3"
    >
      Zeppelin uses a hybrid grid layout as demonstrated below
    </p>
    <div
      class="GridPage-player-0-1-2"
    >
      <div>
        <video
          autoplay=""
          loop=""
          preload="auto"
          src="https://cdn-zel.zepdev.net/zel/animation_grid.mp4"
          style="width: 100%; height: 100%;"
        />
      </div>
    </div>
    <h3
      class="zep-typo--normal-6 GridPage-headline-0-1-3"
    >
      Basic Grid Data
    </h3>
    <div
      class="Table-root-0-1-13"
    >
      <table
        class="Table-table-0-1-14"
      >
        <caption
          class="zep-visually-hidden"
        >
          basic grid data
        </caption>
        <thead
          class="Table-header-0-1-18"
        >
          <tr
            class="Table-tr-0-1-16"
          >
            <th
              class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
              scope="col"
            >
              identifier
            </th>
            <th
              class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
              scope="col"
            >
              screensize
            </th>
            <th
              class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
              scope="col"
            >
              columns
            </th>
            <th
              class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
              scope="col"
            >
              margin
            </th>
            <th
              class="Table-th-0-1-15 zep-typo--normal-1 Table-capitalize-0-1-20"
              scope="col"
            >
              gutter
            </th>
          </tr>
        </thead>
        <tbody
          class="Table-tbody-0-1-17"
        >
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              xxs
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              &lt;576px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              4
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              24px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              24px
            </td>
          </tr>
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              xs
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              ≥576px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              6
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
          </tr>
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              s
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              ≥768px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              6
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              48px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
          </tr>
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              m
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              ≥992px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              8
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              64px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
          </tr>
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              l
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              ≥1200px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              12
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              64px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
          </tr>
          <tr
            class="Table-tr-0-1-16"
          >
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
              scope="row"
            >
              xl
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              ≥1680px
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              12
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              auto
            </td>
            <td
              class="Table-th-0-1-15 Table-borderBottom-0-1-21 zep-typo--normal-2"
            >
              32px
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
